tslint-a11y.js

```diff
@@ -0,0 +1,21 @@
+const path = require("path");
+
+module.exports = {
+  rulesDirectory: path.dirname(require.resolve("tslint-microsoft-contrib")),
+  rules: {
```

Comment:
I tried adding these to [@dcos/tslint-config](https://github.com/dcos-labs/tslint-config/blob/master/index.js) instead of keeping them only in ui-kit, but when I used `npm link` to test it, I got this error:

```
{ Error: Cannot find module 'typescript'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/Users/mikep/tslint-config/node_modules/tslint-microsoft-contrib/reactA11yAnchorsRule.js:20:10)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3) code: 'MODULE_NOT_FOUND' }
```